src/main/scala/com/actian/spark_vector/writer/DataStreamConnector.scala

```diff
@@ -47,7 +50,8 @@ class DataStreamConnector(writeConf: WriteConf) extends Logging with Serializabl
     closeResourceAfterUse(socket) { op(socket) }
   }
 
-  def skipTableInfo(implicit socket: SocketChannel): Unit = {
-    DataStreamReader.readWithByteBuffer { in => } // skip table information message. TODO(): read at least the expected vectorsize
+  def readConnectionHeader(implicit socket: SocketChannel): VectorConnectionHeader = readWithByteBuffer { in =>
```

Comment:
there is more information in this header... (column data types, nullability, etc.). Mention that we ignore it since we get it way before from a JDBC query.